src/server.js

```diff
@@ -259,20 +259,20 @@ class Server {
     const events = this._events
     return Promise.map(tasks.keySeq(), (queue) => {
       return this._rabbitmq.subscribeToQueue(queue, (job, done) => {
-        this._runWorker(queue, tasks.get(queue), job, done)
+        return this._runWorker(queue, tasks.get(queue), job, done)
       })
     })
-      .then(() => {
-        return Promise.map(events.keySeq(), (exchange) => {
-          return this._rabbitmq.subscribeToFanoutExchange(
-            exchange,
-            (job, done) => {
-              this._runWorker(exchange, events.get(exchange), job, done)
-            }
-          )
-        })
+    .then(() => {
```

Comment:
Future .then()'s should not be indented though, since they are part of the same chain. But it makes following subchains easier I think. Personal preference, I assume this passes standard, so we can't argue.